src/datetime/mod.rs

```diff
@@ -28,7 +28,7 @@ use crate::offset::Local;
 use crate::offset::{FixedOffset, Offset, TimeZone, Utc};
 use crate::{Datelike, Months, TimeDelta, Timelike, Weekday};
 
-#[cfg(feature = "rkyv")]
+#[cfg(any(feature = "rkyv_16", feature = "rkyv_32", feature = "rkyv_64"))]
```

Comment:
This seems fairly ugly to spread all over the place. I propose that we instead use an `rkyv-any` (open to better naming suggestions) feature that is enabled by all the size features (add a comment that it is for internal use only, maybe?).

Reply:
Fixed with an internal feature named `__internal-rkyv` since there was already a feature `__internal-bench`.